src/services/models/logEventModels.js

```diff
@@ -2,11 +2,23 @@
 
 import { toDiagnosticsModel } from 'services/models';
 
-export const toLogRuleModel = (rule = {}) => ({
-  'DeviceGroup': rule.groupId,
-  'Calculation': rule.calculation,
-  'TimePeriod': rule.timePeriod,
-  'Severity': rule.severity
-});
-
-export const toRuleDiagnosticsModel = (eventName, rule) => toDiagnosticsModel(eventName, toLogRuleModel(rule));
+export const toRuleDiagnosticsModel = (eventName, rule) =>
+{
+  var metadata = {
+    DeviceGroup: rule.groupId,
+    Calculation : rule.calculation,
+    TimePeriod: rule.timePeriod,
+    SeverityLevel: rule.severity,
+    ConditionCount: rule.conditions.length,
+    FirstFieldChosen: rule.conditions[0].field,
+    FirstOperatorChosen: rule.conditions[0].operator
+  };
+
+  return toDiagnosticsModel(eventName, metadata);
+}
+
+export const toSinglePropertyDiagnosticsModel = (eventName, propertyTitle, property) => {
+  var metadata = {};
```

Comment:
@spryor is there a shortcut for this two line assignment?

Reply:
```js
const metadata = { propertyTitle: property };
```